app/models/host_initiator.rb

```diff
@@ -16,6 +16,8 @@ class HostInitiator < ApplicationRecord
   virtual_total :v_total_addresses, :san_addresses
 
   supports_not :create
+  supports_not :delete
```

Comment:
this line is not needed

can you also delete the create above it? (if not, then I can sweep back later)

Reply:
Sure, I will remove it👍